frontend/src/metabase/query_builder/actions/core/card.ts

```diff
@@ -1,10 +1,9 @@
 import Questions from "metabase/entities/questions";
-import type { CardId } from "metabase-types/api";
 import type { Dispatch, GetState } from "metabase-types/store";
 
 // load a card either by ID or from a base64 serialization.  if both are present then they are merged, which the serialized version taking precedence
 export async function loadCard(
-  cardId: CardId,
```

Comment:
We can't simply change the `CardId` type into `string | number` as that breaks lots of querying stuff. Let's keep the change localized for now.